lib/workers/repository/init/index.ts

```diff
@@ -21,6 +22,7 @@ export async function initRepo(
   config = await initApis(config);
   config = await getRepoConfig(config);
   checkIfConfigured(config);
+  config = applyEnabledManagersFilter(config);
```

Comment:
> shouldn't it disable all other managers? Are you sure the manager config is always there. We need to be sure it disables all other manager even there is no manager config yet.

@viceice It seemed to me that all relevant data is ready at this point, isn't it?

Reply:
I'm not sure 🤷‍♂️